lib/config/default.js

```diff
@@ -56,6 +56,8 @@ module.exports = {
   // socket.io
   heartbeatInterval: 5000,
   heartbeatTimeout: 10000,
+  // too busy timeout
+  tooBusyLag: undefined,
```

Comment:
I think we should add 70 in this place.

* It's more self-documenting
* If the default was ever changed (no, I'm not assuming that it does), this will not get out of with our readme.